templates/careers/progression.html

```diff
@@ -1,32 +1,343 @@
+      <img class="p-collage--2" data-animation="u-animation--slide-from-bottom" src="https://assets.ubuntu.com/v1/ae1b866a-Image_2.jpg" alt="Employees working together on same laptop" style="animation-delay: 0.25s"/>
+      <img class="p-collage--3" data-animation="u-animation--slide-from-bottom" src="https://assets.ubuntu.com/v1/90651b21-Image_3.png" alt="Employees working together on different laptops" style="animation-delay: 0.50s" />
+      <img class="p-collage--4" data-animation="u-animation--slide-from-bottom" src="https://assets.ubuntu.com/v1/c707c1cb-Image 4.png" alt="Employee looking at her coworker while smiling" style="animation-delay: 0.75s" />
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--careers-progression p-careers-progression__dividers">
+  <div class="row" style="border-left: 1px solid rgba(255, 255, 255, 0.25); padding-left: 1.375rem;">
+    <div class="col-4 col-medium-3" data-animation="u-animation--slide-from-bottom">
+      <div class="p-roles">
+        <div>
+          <h2>Explore diverse roles</h2>
+          <p>We encourage team members to experience diverse roles at Canonical over the course of their career to build deep awareness of company culture,  establish best practices in leadership and grow their technical expertise.</p>
+          <p>In fact, flexibility in exploring new roles is one of main reasons people love to work here.</p>
+        </div>
+        <div class="p-strip--careers-progression u-no-padding--bottom">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/e8636167-diverse-roles.png",
+            alt="",
+            width="324",
+            height="487",
+            hi_def=True,
+            loading="lazy",
+            attrs={"style": "width: 100%"}
+            ) | safe
+          }}
         </div>
       </div>
     </div>
-  </section>
-{% endblock %}
+    <div class="col-4 col-medium-3" style="animation-delay: 0.25s" data-animation="u-animation--slide-from-bottom">
+      <div class="p-roles">
+        <div>
+          <h2>Finding your passion</h2>
+          <p>We also explicitly encourage team members to find their passion and invest in that, recognising the benefit to your team and the company when you become a more effective team player.</p>
+        </div>
+        <div class="p-strip--careers-progression u-no-padding--bottom">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/e0b9fe6e-finding-your-passion.png",
+            alt="",
+            width="324",
+            height="487",
+            hi_def=True,
+            loading="lazy",
+            attrs={"style": "width: 100%"}
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3" data-animation="u-animation--slide-from-bottom" style="animation-delay: 0.50s">
+      <div class="p-roles" style="border-right: 0;">
```

Comment:
This online style can be more elegantly handled in CSS `last-of-type`

Reply:
So I think the trade-off is that it will become more messy in the stylesheet. It's not exactly `last-child` it's actually last-child of a different parent i.e. a "cousin". So to use a selector for the cousin means I will need to create a parent selector (new parent selector? Because I'm using Vanilla styles mostly here, no custom parent class) that goes down to the child, just so I can add one line of CSS.